### DISCUSSION TITLE
Google Summer of Code Ideas

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1584] [General]
Mesa is applying to be part of the Google Summer of Code under NUMFOCUS. This discussion is to generate ideas so we can add them to our [Wikipage](https://github.com/projectmesa/mesa/wiki/Projects-for-Google-Summer-of-Code-2023) **prior to Feb 5th.** 

NUMFOCUS idea last year - https://github.com/numfocus/gsoc/blob/master/2022/ideas-list.md

Here's some GSoC 2022 templates I found helpful, that we could model our wikipage after: 
cupy - https://github.com/cupy/cupy/wiki/GSoC-2022-Project-Ideas
arviz - https://github.com/arviz-devs/arviz/wiki/GSoC-2022-projects
jump-dev - https://github.com/jump-dev/GSOC2022

**Comment by rht** [comment]
The NetworkX one is also good: https://networkx.org/documentation/latest/developer/projects.html.
